### PR TITLE
Add StepDownPrimary to the replicaset methods.

### PR DESCRIPTION
**Pull request by jameinel**
This will ask the Primary to step down using the default parameters (60s
the primary should not become primary again, and 10s for a secondary to
become up-to-date, and not forcing the step down if a secondary fails to
get up to date.)

We also suppress the io.EOF because we expect all sessions to get
disconnected. Anything calling this function needs to be prepared for
that to happen, but it is expected, not an error.

### CODE REVIEW

**Comment by jameinel** [comment]
[replicaset StepDownPrimary](https://trello.com/c/xXdI7Y1N/168-replicaset-stepdownprimary)

File: replicaset.go [hunk 1]
```diff
@@ -421,6 +421,27 @@ type Config struct {
...
+// but all other sessions will also be disconnected.
+func StepDownPrimary(session *mgo.Session) (error) {
+       strictSession := session.Clone()
+       defer strictSession.Close()
+       // StepDown can only be called on the primary
+       session.SetMode(mgo.Primary, true)
+       // replSetStepDown can take a number of arguments, such as "number of seconds to wait for a secondary to become
+       // up-to-date", "how long to force the current primary to not be primary", and "force" whether or not the primary
+       // should be demoted right now. But we just use the defaults for all of those.
+	   err := session.Run("replSetStepDown", nil)
```
Comment (manadart) [inline replicaset.go]: Rest of the method has an odd 3-space indent...

**Review by manadart — APPROVED** [review]

File: replicaset.go (hunk 1 above)
Comment (jameinel) [reply replicaset.go]: thanks, just doing this on a different machine and forgot to run go fmt.

**Comment by jameinel** [comment]
$$merge$$

**Comment by jameinel** [comment]
$$merge$$ the bot was configured without mongo being installed.

**Comment by jameinel** [comment]
$$merge$$ I pushed up a small change. It seems 2.4 and 3.2 have slightly different behaviors (3.4 and 3.6 seem to follow 3.2 behavior.)

**Pull request merged**